altair/vegalite/v5/schema/_config.py

```diff
@@ -0,0 +1,1055 @@
+    timeUnitBandSize: float
+    tooltip: str | bool | None | float | TooltipContent
+    url: str
+    width: float
+    x: str | float
+    x2: str | float
+    y: str | float
+    y2: str | float
+
+
+class ProjectionConfigKwds(TypedDict, total=False):
+    """Placeholder doc."""
+
+    center: Sequence[float]
+    clipAngle: float
+    clipExtent: Sequence[Sequence[float]]
+    coefficient: float
+    distance: float
+    extent: Sequence[Sequence[float]]
+    fit: (
+        GeoJsonFeature
+        | GeoJsonFeatureCollection
+        | Sequence[GeoJsonFeature]
+        | Sequence[GeoJsonFeature | GeoJsonFeatureCollection | Sequence[GeoJsonFeature]]
+    )
+    fraction: float
+    lobes: float
+    parallel: float
+    parallels: Sequence[float]
+    pointRadius: float
+    precision: float
+    radius: float
+    ratio: float
+    reflectX: bool
+    reflectY: bool
+    rotate: Sequence[float]
+    scale: float
+    size: Sequence[float]
+    spacing: float
+    tilt: float
+    translate: Sequence[float]
+    type: ProjectionType_T
+
+
+class RangeConfigKwds(TypedDict, total=False):
+    """Placeholder doc."""
+
+    category: (
+        Sequence[str | ColorName_T]
+        | Sequence[str | bool | None | float | Sequence[float]]
+        | RangeEnum_T
+    )
+    diverging: (
+        Sequence[str | ColorName_T]
+        | Sequence[str | bool | None | float | Sequence[float]]
+        | RangeEnum_T
+    )
+    heatmap: (
+        Sequence[str | ColorName_T]
+        | Sequence[str | bool | None | float | Sequence[float]]
+        | RangeEnum_T
+    )
+    ordinal: (
+        Sequence[str | ColorName_T]
+        | Sequence[str | bool | None | float | Sequence[float]]
+        | RangeEnum_T
+    )
+    ramp: (
+        Sequence[str | ColorName_T]
+        | Sequence[str | bool | None | float | Sequence[float]]
+        | RangeEnum_T
+    )
+    symbol: Sequence[str | SymbolShape]
+
+
+class ScaleConfigKwds(TypedDict, total=False):
+    """Placeholder doc."""
+
+    bandPaddingInner: float
+    bandPaddingOuter: float
+    bandWithNestedOffsetPaddingInner: float
+    bandWithNestedOffsetPaddingOuter: float
+    barBandPaddingInner: float
+    clamp: bool
+    continuousPadding: float
+    invalid: ScaleInvalidDataConfigKwds
+    maxBandSize: float
+    maxFontSize: float
+    maxOpacity: float
+    maxSize: float
+    maxStrokeWidth: float
+    minBandSize: float
+    minFontSize: float
+    minOpacity: float
+    minSize: float
+    minStrokeWidth: float
+    offsetBandPaddingInner: float
+    offsetBandPaddingOuter: float
+    pointPadding: float
+    quantileCount: float
+    quantizeCount: float
+    rectBandPaddingInner: float
+    round: bool
+    tickBandPaddingInner: float
+    useUnaggregatedDomain: bool
+    xReverse: bool
+    zero: bool
+
+
+class SelectionConfigKwds(TypedDict, total=False):
+    """Placeholder doc."""
+
+    interval: IntervalSelectionConfigWithoutType
+    point: PointSelectionConfigWithoutType
```

Comment:
Think both of these need `TypedDict`, but were missed since they don't end with `Config`